packages/stylelint-plugin/src/rules/no-invalid-use-of-var-tokens-in-equations.spec.ts

```diff
@@ -14,6 +16,7 @@ describe("no-invalid-use-of-var-tokens-in-equations rule", () => {
         },
       }
     )
+    /* eslint-disable-next-line jest/valid-expect */
     return expect(reported === 0)
```

Comment:
Wait, we're returning an expect? 😵

Reply:
If you expand this, you'll see it's essentially a custom `expect` that's used further down.
The spec is way too wild, I'm not refactoring it (in this PR).